Assets/_Scripts/PlayerBuffStatus.cs

```diff
@@ -8,8 +8,8 @@ public class PlayerBuffStatus : MonoBehaviour {
 //	private List<bool> Buff_valid_List;	// keep track of the all the buff which is valid 
 
 
-	private bool IncreaseNumber = false;
-	private bool Bigger = false;
+	public bool IncreaseNumber = false;
```

Comment:
You may provide get accessor (Property) instead of just public.
refer to this 
https://msdn.microsoft.com/en-us/library/x9fsa0sw.aspx